.buildkite/x-pack/pipeline.xpack.packetbeat.yml

```diff
@@ -18,54 +17,274 @@ env:
+
+      - label: ":windows: Windows 2019 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "extended-win-2019-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2019}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2019 Unit Tests"
+
+      - label: ":windows: Windows 10 System Tests"
+        key: "extended-win-10-system-tests"
+        skip: "skipping due to elastic/beats#38142"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage systemTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_10}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2022 System Tests"
+
+  - group: "x-pack/packetbeat Linux arm Extended Tests"
+    key: "x-pack-packetbeat-extended-linux-arm-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+    steps:
+      - label: ":linux: Ubuntu ARM Unit Tests"
+        key: "extended-arm64-unit-test"
+        command: "cd x-pack/packetbeat && mage build unitTest"
+        if: build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Ubuntu ARM Unit Tests"
+
+  - group: "x-pack/packetbeat MacOS Extended Tests"
+    key: "x-pack-packetbeat-extended-macos-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macos.*/
+    steps:
+      - label: ":mac: MacOS Unit Tests"
+        key: "extended-macos-unit-tests"
+        command: ".buildkite/scripts/unit_tests.sh"
+        agents:
+          provider: "orka"
+          imagePrefix: "${IMAGE_MACOS_X86_64}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: MacOS Unit Tests"
+
+      - label: ":mac: MacOS arm64 Unit Tests"
+        key: "macos-arm64-unit-tests-extended"
+        command: ".buildkite/scripts/macos_unit_tests.sh"
```

Comment:
I don't think this script exists in [main](https://github.com/elastic/beats/tree/main/.buildkite/scripts)? 

In any case, I suggest we replace it with the above snippet i.e.

```
          set -euo pipefail
          source .buildkite/scripts/install_macos_tools.sh
          cd heartbeat
          mage build unitTest
```

as done for [other pipelines](https://github.com/elastic/beats/blob/main/.buildkite/heartbeat/heartbeat-pipeline.yml#L150).